src/shell/log.c

```diff
@@ -282,11 +283,29 @@ void flux_shell_fatal (const char *component,
     char buf [4096];
     va_list ap;
 
+    if (logger.fatal_in_progress) {
+        if (shell->verbose > 0) {
```

Comment:
Suggestion: just log the second fatal message so we don't lose any possible debug/error information, but return early before invoking other fatal error actions.

Reply:
the reason I did this was b/c something as simple as `flux run nosuchcommand` would trigger two fatal error messages.  Think that's ok?  Or perhaps I can adjust the second fatal output message to indicate "maybe not the immportant one" in some way.

Reply:
Oh, ugh :frowning_face: Yeah, I guess suppressing the error is best then.